react/components/ProductImages/components/Carousel/Loader.js

```diff
@@ -2,6 +2,40 @@ import ContentLoader from 'react-content-loader'
 import React from 'react'
 
 export default props => {
+  if (props.slidesAmount === 0) {
+    return (
+      <svg
```

Comment:
I think it would be better to place this SVG in another file.

Does it make sense to be in `store-icons`? Or that's far fetched?

Reply:
It can be in another file. I thought it would be too much, but if you see it that way, no problem in changing it.
I don't know if it's better to be a `store-icon` because it feels like an image, to me. Really don't know if it's semantically wrong to put it like this.

Reply:
About creating another file, I think it would just make it more clearer, since you would have to give it a name.

Example:

```jsx
import ImagePlaceholder from './ImagePlaceholder'

// ...

if (props.slidesAmount === 0) {
  return <ImagePlaceholder />
}
```

Reply:
You could also mock this in tests, since the real SVG doesnt make a difference in snapshots

Reply:
Cool. I'll do it :)

Reply:
Than it could appear as something like this in snapshots

```jsx
<div>Mocked Image Placeholder</div>
```